Code/Engine/GameEngine/Animation/Implementation/PropertyAnimComponent.cpp

```diff
@@ -122,7 +122,6 @@ void ezPropertyAnimComponent::PlayAnimationRange(ezTime RangeLow, ezTime RangeHi
   m_AnimationRangeLow = RangeLow;
   m_AnimationRangeHigh = RangeHigh;
 
-  m_AnimationTime = m_AnimationRangeLow;
```

Comment:
Don't set animation time here since animation range low is not yet clamped against the actual anim duration.